### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by codecaiine**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+      </div>
+    </div>
+    <!--Pricing table ends-->
+    <footer class="pt-4 my-md-5 pt-md-5 border-top" style="padding-left: 30px; padding-right: 30px;">
+      <div class="row">
+        <div class="col-12" style="text-align: right;">
+          <h5><b>Privacy</b> &nbsp;&nbsp;&nbsp; <b>Terms</b> &nbsp;&nbsp;&nbsp;<b>Contract</b></h5>
+          Copyright 2016, Original Trombones
+        </div>
+
```
Comment (codecaiine) [inline index.html:187]: This space doesn't look good

**Review by codecaiine — COMMENTED** [review]
This work looks good. 
Well done.

However, You need to make some changes. 

Happy coding

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @codecaiine🔥 

You are code reviewing like a Pro, nothing more to add here, good job!👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codecaiine** [comment]
Thank you @mricanho .

**Pull request merged**